vermouth/tests/test_ff_files.py

```diff
@@ -1041,7 +1057,7 @@ def test_variables():
     del ff.variables['float']
     assert ff.variables == {'integer': 1, 'string': 'mass',
                             'dictionary': {'a': 1, 'b': {'plop': 'toto'}}}
-
+test_variables()
```

Comment:
This looks like it doesn't belong here

Reply:
why not?

Reply:
pytest invokes test methods. So there's absolutely no need to do so by hand.